src/Storage/Bucket.php

```diff
@@ -801,8 +801,6 @@ public function isWritable($file = null)
     /**
      * Manage the IAM policy for the current Bucket.
      *
-     * Please note that this method may not yet be available in your project.
```

Comment:
@jdpedrie I have remove the comment, as this should now be available - what else do I need to update for whitelist CI testing?